squirrels/_api_server.py

```
@@ -228,7 +228,8 @@ def _print_banner(self, mount_path: str, host: str | None, port: int | None, is_
         print(" 🖥️  Application UI")
         print(f"  └─ Squirrels Studio: {full_hostname}{mount_path_stripped}/studio")
         if show_multiple_options:
-            print(f"     └─ (The following URL also redirects to studio: {full_hostname})")
+            print(f"     ├─ The root path also redirects to Squirrels Studio: {full_hostname}/")
+        print(    f"     └─ This requires an internet connection to load the JS and CSS files")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)